apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktBarnService.kt

```diff
@@ -17,11 +17,15 @@ internal class DoedshendelseKontrollpunktBarnService(
         avdoed: PersonDTO,
         sak: Sak?,
         barn: PersonDTO,
-    ): List<DoedshendelseKontrollpunkt> =
-        listOfNotNull(
-            kontrollerBarnOgHarBP(sak),
-            kontrollerSamtidigDoedsfall(avdoed, hendelse, barn),
-        )
+        kontrollerSamtidigDoedsfall: Boolean = true,
+    ): List<DoedshendelseKontrollpunkt> {
+        val kontrollpunkt = mutableListOf<DoedshendelseKontrollpunkt>()
+        kontrollerBarnOgHarBP(sak)?.let { kontrollpunkt.add(it) }
+        if (kontrollerSamtidigDoedsfall) {
+            kontrollerSamtidigDoedsfall(avdoed, hendelse, barn)?.let { kontrollpunkt.add(it) }
```

Comment:
Kan du bare fjerne det for alle?

Reply:
Nei, det skal jo brukes videre for det som ikke er en del av denne midlertidige jobben